### PR TITLE
Feature/validation for add patient vm

### PR DESCRIPTION
**Pull request by KostetskyiYaroslav**

### CODE REVIEW

File: MedicalDocManagment.WebUI/Models/Validators/AddPatientValidator.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+{
+    public class AddPatientVMValidator : AbstractValidator<AddPatientVM>
+    {
+        public AddPatientVMValidator()
+        {
+            RuleSet("Names", () =>
+            {
+                RuleFor(patient => patient.FirstName)
+                       .Matches(@"[А-Яа-яґҐєЄіІїЇ`´ʼ’ʼ’'-]{1,100}")
+                       .WithMessage("Ім'я пацієнта написанне не кирилицею");
```
Comment (AndriiKatsubo) [inline MedicalDocManagment.WebUI/Models/Validators/AddPatientValidator.cs]: "Написане"

File: MedicalDocManagment.WebUI/Models/Validators/AddPatientValidator.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                       .WithMessage("Дата народження пацієнта є більшою за сьогоднішню");
+            });
+
+            RuleFor(patient => patient.Address)
+                   .Matches(@"[А-Яа-яґҐєЄіІїЇ`´ʼ’ʼ’'-]{1,100}")
+                   .WithMessage("Адресу пацієнта вказано невірно");
+
+            RuleFor(patient => patient.DiagnosisCode)
+                   .Matches(@"[A-Z]\d{2}.\d")
+                   .WithMessage("Діагноз для пацієнта виставленно невірно");
```
Comment (AndriiKatsubo) [inline MedicalDocManagment.WebUI/Models/Validators/AddPatientValidator.cs]: "Виставлено"

**Review by AndriiKatsubo — DISMISSED** [review]
Requested grammar mistakes fixing.

**Review by Ravennod — APPROVED** [review]

**Comment by AndriiKatsubo** [comment]
default
Should this request be validated?

**Comment by AndriiKatsubo** [comment]
I think that request with no diagnosis shouldn't be validated.
default

**Review by AndriiKatsubo — APPROVED** [review]

**Pull request merged**